manifests/server.pp

```diff
@@ -190,24 +190,27 @@
   $conf_file_name = "redis_${redis_name}.conf"
   $conf_file = "/etc/${conf_file_name}"
   file { $conf_file:
-      ensure  => file,
-      content => template('redis/etc/redis.conf.erb'),
-      require => Class['redis::install'];
+    ensure  => file,
+    content => template('redis/etc/redis.conf.erb'),
+    require => Class['redis::install'];
   }
 
   # startup script
-  case $::osfamily {
+  case $::operatingsystem {
```

Comment:
"Better" solution would be, to have a seperate sub case in the Debian part. And have the operatingsystem specific part there.